### PR TITLE
goquiet 1.0.2 (new formula)

### PR DESCRIPTION
**Pull request by icpz**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

File: Formula/goquiet.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    (buildpath/"src/github.com/cbeuw/GoQuiet").install buildpath.children
+
+    cd "src/github.com/cbeuw/GoQuiet" do
+      system "make"
+      bin.install Dir["build/*"]
```
Comment (DomT4) [inline Formula/goquiet.rb]: You'll likely need to add a `prefix.install_metafiles` line here.

This ensures things like `README`, `LICENSE` and `CHANGELOG` where available are installed into the prefix. Normally Homebrew handles this automatically with a little magic, but when you relocate the entire contents of the `buildpath` to another directory for the install process as is common with `go`-using formulae that logic falls on its face & it needs a little manual nudging. It's a known issue.

Comment (lcdtyph) [reply Formula/goquiet.rb]: Done, thank you for the review!

File: Formula/goquiet.rb [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+
+class Goquiet < Formula
+  desc "Shadowsocks plugin that obfuscates the traffic"
+  homepage "https://github.com/cbeuw/GoQuiet"
+  url "https://github.com/cbeuw/GoQuiet/archive/v1.1.1.tar.gz"
+  sha256 "55f92df1dcf892d7aaccd0331d917a41286dafb5a7d8fa7b8d2d257dc385314f"
+
+  depends_on "go" => :build
+
+  go_resource "github.com/cbeuw/gotfo" do
```
Comment (commitay) [inline Formula/goquiet.rb]: > `go_resource`s are deprecated. Please ask upstream to implement Go vendoring

**Pull request closed without merging**